caproto/_data.py

```diff
@@ -249,6 +271,7 @@ def get_dbr_data(self, type_):
     async def set_dbr_data(self, data, data_type, metadata):
         '''Set data from DBR metadata/values'''
         self.value = self.fromtype(values=data, data_type=data_type)
+        self.timestamp = time.time()
```

Comment:
Any thoughts on how/whether we should ensure that this timestamp is consistent with the timestamp optionally passed in through `metadata`?

Reply:
(I'm addressing this in #90.)